drivers/serial/uart_ns16550.c

```diff
@@ -1283,7 +1292,7 @@ static const struct uart_driver_api uart_ns16550_driver_api = {
 	};                                                                           \
 	DEVICE_DT_INST_DEFINE(n, &uart_ns16550_init, NULL,                           \
 			      &uart_ns16550_dev_data_##n, &uart_ns16550_dev_cfg_##n, \
-			      PRE_KERNEL_1, CONFIG_SERIAL_INIT_PRIORITY,             \
+			       BOOT_LEVEL(n), CONFIG_SERIAL_INIT_PRIORITY,             \
```

Comment:
Looks like the alignment is different after this change - you've added one more space